doc/contributing/submitting-changes.chapter.md

```diff
@@ -83,6 +83,62 @@ If a security fix applies to both master and a stable release then, similar to r
 
 Critical security fixes may by-pass the staging branches and be delivered directly to release branches such as `master` and `release-*`.
 
+## Writing package tests {#submitting-changes-package-tests}
```

Comment:
Perhaps this should go to the coding conventions rather than submitting changes chapter?

Reply:
i think that is worse

there is not really a good place. all documentation should be restructured at some point (when markdown migration is finished)

it might be good to have it in the same chapter "Testing" with Nixos tests, but that is an a separate manual...

i just want to have the information there and it can be improved later

Reply:
The “Coding conventions” name is not the most fitting for Chapter 18 but looking at its contents, it is pretty clear stuff like this should go there – the whole chapter concerns itself with writing package expressions. On the other hand Chapter 19 (Submitting changes) mainly covers GitHub etiquette.

Reply:
yeah, the chapter name is not fitting. you are right that it fits to the other content. i moved it. thanks!